hunter_base/launch/hunter_base.launch.py

```diff
@@ -27,6 +27,17 @@ def generate_launch_description():
     sim_control_rate_arg = DeclareLaunchArgument('control_rate', default_value='50',
                                                  description='Simulation control loop update rate')
     
+    kp_v = LaunchConfiguration('kp_v', default='40.0')
```

Comment:
These parameters definitely need documentation. It's not clear what any of these actually do.